Directory.Build.targets

```diff
@@ -57,5 +57,101 @@
 
   </Target>
 
+  <!-- For build debugging purposes. -->
+  <Target Name="Vars">
```

Comment:
Nit: We have this in another place as well, should we put this in a shared `.targets` file?

Reply:
It was originally placed in `vnext/Directory.Build.targets`.

I moved it to the root TARGETS file so this diagnostic-only target is available for any project in the repository.

Reply:
Thanks, noticed later int he review that it was a move :)